src/steps/account/__recordings__/fetchAccountDetailsShouldCollectData_2107956155/recording.har

```diff
@@ -0,0 +1,139 @@
+        "_order": 0,
+        "cache": {},
+        "request": {
+          "bodySize": 11,
+          "cookies": [],
+          "headers": [
+            {
+              "name": "user-agent",
+              "value": "got (https://github.com/sindresorhus/got)"
+            },
+            {
+              "name": "x-mc-date",
+              "value": "Tue, 11 Jan 2022 19:25:06 UTC"
+            },
+            {
+              "name": "x-mc-req-id",
+              "value": "6cdb9a70-2ac3-45c5-857e-9b6cf1d9b24c"
+            },
+            {
+              "name": "x-mc-app-id",
+              "value": "[REDACTED]"
+            },
+            {
+              "name": "authorization",
+              "value": "[REDACTED]"
+            },
+            {
+              "name": "content-length",
+              "value": "11"
+            },
+            {
+              "name": "accept-encoding",
+              "value": "gzip, deflate, br"
+            },
+            {
+              "name": "host",
+              "value": "us-api.mimecast.com"
+            }
+          ],
+          "headersSize": 571,
+          "httpVersion": "HTTP/1.1",
+          "method": "POST",
+          "postData": {
+            "mimeType": "text/plain",
+            "params": [],
+            "text": "{\"data\":[]}"
+          },
+          "queryString": [],
+          "url": "https://us-api.mimecast.com/api/account/get-account"
+        },
+        "response": {
+          "bodySize": 671,
+          "content": {
+            "_isBinary": true,
+            "mimeType": "application/json",
+            "size": 671,
+            "text": "[\"1f8b08000000000000009554514fdb3010fe2b969f36a9a0a4a530fab4a854532718555bf652f5e170aead476247b653a810ff9d3ba781306d93f69238beefeebeef3b3bcfb2c40072f42c7d80507b39ea27c94b4fe6c0bbab67e970abad912349b19e04a7767a8f72b481c2634f6e21e0231ce428b89a3e2bf0beda39f0849004f775555917c636e78d497a91d266094f730c6842ac3b483eee8cadd9685762ded62cb5790b4e0cdc171c3ab6873ad89228e40bdc960499d56e8b6d623854dc7553170575ad6ca1d5616a76e87400a3de35b0d47ba27c6459fb80aebc64f6c872e7089e891ed1ca1a2693e5c44bfbe0205877851ba88bf053e32315b82143a37dc48f510bf49eb82f7589b60e72744e8af1897dc92afd2e45295b9bd6aaf1dd221b0cb3b36420df423fa0e4d0ec762c4e44369bd2f37b5d69627b6bb06d362941178df9cc145468773c94a7607257fbafbf9a346bf054d992a039b9a84d93567b7463ee274729cf86682bf0614aaecb243d4992c1901e0db30ad4036c914ecd8a647b4f6b314765f7e80e62816eaf1512cd3b2a29566932fcb2a6a471e3a0e0b7b3852052e28afc12d7080f62e670df0c5b7c5a10cbcf9c980efe2f71e66c9337e4bc29a5390385885610d0065481e3e7671c6f0726968ef4c4b2bcc20e81419f814dfec42877a8622386ce9cde833a74c0fdc8360be4fe8e4f65dbf1a3a6e1f93f50ad80e16504e57b3eb1b9b859669d0a4924b540553b3c6afbd6dcc72ef358e16e7efd6716670d57ba4734385f5993a3f3dd1ec3c6a1bfccb6d3278dc0e36911b302c2c6ba3286628f05fd5fa83767a7a7499742ff37236ec050b3b8ec1089a39a9615f1b32696ea2ae261268c58ee1c4220a6f40bd1d0c62ea28cec111c1ad2c2f3a5eb6bb63146a772fdb2eec94dbc28abf5cb2b07cc0fe415050000\"]"
```

Comment:
This is gzipped, can we decompress this? See: https://github.com/JupiterOne/graph-sysdig/blob/a72be1b8854bb18f9c6825b1e11feb36708185b6/test/recording.ts#L12